src/SFA.DAS.EmployerCommitmentsV2.Web/Mappers/Cohort/CohortsSummaryViewModelMapper.cs

```diff
@@ -43,7 +43,7 @@ public async Task<CohortsViewModel> Map(CohortsByAccountRequest source)
                     "with transfer sending employers",
                     _linkGenerator.CommitmentsV2Link($"{source.AccountHashedId}/unapproved/with-transfer-sender")),
                 
-                BackLink = _linkGenerator.CommitmentsLink($"accounts/{source.AccountHashedId}/apprentices/home")
+                BackLink = _linkGenerator.CommitmentsV2Link($"{source.AccountHashedId}/unapproved")
```

Comment:
As it is redirecting with in V2, I think Url.Action is better. But as this is inside mapper, Url may not be available in it.

I will leave it up to you to decide, whether it is too much work for this.